### PR TITLE
Modernize integration tests 2024-04-23

### PR DESCRIPTION
**Pull request by chapman39**
This PR modernizes the integration tests (instead of rebaselining all the tests) and will be replacing my previous one: https://github.com/LLNL/serac_tests/pull/20 

- [x] Update the curve name changes and add new curve variables as needed
- [x] Remove tests that need to be fixed
- [x] Rebaseline tests that failed

### CODE REVIEW

**Comment by btalamini** [comment]
Thanks, it will be easier to see whats going on this way.

**Comment by chapman39** [comment]
@btalamini All that's left now are the following tests that fail the tolerance test:

- dyn_direct_solve_serial
- dyn_direct_solve_parallel
- dyn_solve_serial
- dyn_solve_parallel
- dyn_exp_solve_serial
- dyn_exp_solve_parallel
- static_solve_serial
- static_solve_parallel

This is the full log for `dyn_direct_solve_serial`:

```
#ATS:commandLine =/usr/WS2/meemee/serac/repo/scripts/testing/tolerance_test.py --baseline=/usr/WS2/meemee/serac/repo/tests/integration/solid/dyn_direct_solve/dyn_direct_solve_serial.json --test=/usr/WS2/meemee/serac/repo/tests/integration/solid/dyn_direct_solve/dyn_direct_solve_serial_output/summary.json --tolerance=1e-05
#ATS:directory =/usr/WS2/meemee/serac/repo/tests/integration/solid/dyn_direct_solve
#ATS:executable =/usr/WS2/meemee/serac/repo/scripts/testing/tolerance_test.py
#ATS:name =tolerance_test(dyn_direct_solve_serial)
#ATS:clas =--baseline=/usr/WS2/meemee/serac/repo/tests/integration/solid/dyn_direct_solve/dyn_direct_solve_serial.json --test=/usr/WS2/meemee/serac/repo/tests/integration/solid/dyn_direct_solve/dyn_direct_solve_serial_output/summary.json --tolerance=1e-05
#ATS:np =1
#ATS:script =
#ATS:testpath =

------- Given Options -------
Baseline file:  /usr/WS2/meemee/serac/repo/tests/integration/solid/dyn_direct_solve/dyn_direct_solve_serial.json
Test file:      /usr/WS2/meemee/serac/repo/tests/integration/solid/dyn_direct_solve/dyn_direct_solve_serial_output/summary.json
Tolerance:      1e-05
Tolerance file: None
-----------------------------
Tolerance for serac_velocity = 1e-05
ERROR: Test value 'serac_velocity/l1norms[0]' out of tolerance: baseline value=0.621399556473554, test value=0.621999629786774
ERROR: Test value 'serac_velocity/l1norms[1]' out of tolerance: baseline value=0.614795932222138, test value=0.614747291346913
ERROR: Test value 'serac_velocity/l1norms[2]' out of tolerance: baseline value=0.59299095278431, test value=0.594428354602046
ERROR: Test value 'serac_velocity/l1norms[3]' out of tolerance: baseline value=0.578827596794006, test value=0.578976956372166
ERROR: Test value 'serac_velocity/l1norms[4]' out of tolerance: baseline value=0.567787422727583, test value=0.569197013912972
ERROR: Test value 'serac_velocity/l1norms[5]' out of tolerance: baseline value=0.565898610602028, test value=0.565753607756741
ERROR: Test value 'serac_velocity/l2norms[0]' out of tolerance: baseline value=0.250112096686039, test value=0.250245497817819
ERROR: Test value 'serac_velocity/l2norms[2]' out of tolerance: baseline value=0.239304790364463, test value=0.239631673775797
ERROR: Test value 'serac_velocity/l2norms[3]' out of tolerance: baseline value=0.235311983503356, test value=0.235369694572078
ERROR: Test value 'serac_velocity/l2norms[4]' out of tolerance: baseline value=0.229532681027992, test value=0.229862309369683
ERROR: Test value 'serac_velocity/linfnorms[1]' out of tolerance: baseline value=0.13374699591133, test value=0.13377165906929
ERROR: Test value 'serac_velocity/linfnorms[3]' out of tolerance: baseline value=0.129995331399174, test value=0.13002932045165
ERROR: Test value 'serac_velocity/linfnorms[4]' out of tolerance: baseline value=0.124772623879401, test value=0.124784017156824
ERROR: Test value 'serac_velocity/linfnorms[5]' out of tolerance: baseline value=0.118528281370322, test value=0.118566551074239
ERROR: Test value 'serac_velocity/mins[2]' out of tolerance: baseline value=-0.0749995195674547, test value=-0.0751508635712267
ERROR: Test value 'serac_velocity/mins[3]' out of tolerance: baseline value=-0.048818685478668, test value=-0.0488401631210627
ERROR: Test value 'serac_velocity/mins[4]' out of tolerance: baseline value=-0.0552958348610414, test value=-0.0552445167443998
ERROR: Test value 'serac_velocity/mins[5]' out of tolerance: baseline value=-0.0720075318598361, test value=-0.0719611688912459
ERROR: Test value 'serac_velocity/maxs[3]' out of tolerance: baseline value=0.121623427176008, test value=0.121644506516159
ERROR: Test value 'serac_velocity/maxs[5]' out of tolerance: baseline value=0.114719415578794, test value=0.114805280971394
Tolerance for serac_displacement = 1e-05
ERROR: Test value 'serac_displacement/l1norms[0]' out of tolerance: baseline value=0.623531359162929, test value=0.624133445685589
ERROR: Test value 'serac_displacement/l1norms[1]' out of tolerance: baseline value=1.23959839064004, test value=1.23942817422266
ERROR: Test value 'serac_displacement/l1norms[2]' out of tolerance: baseline value=1.83600745073608, test value=1.8404016044292
ERROR: Test value 'serac_displacement/l1norms[3]' out of tolerance: baseline value=2.41940173325206, test value=2.42009648525954
ERROR: Test value 'serac_displacement/l1norms[4]' out of tolerance: baseline value=2.97466943548475, test value=2.98253852941844
ERROR: Test value 'serac_displacement/l1norms[5]' out of tolerance: baseline value=3.53404052315755, test value=3.53380811798154
ERROR: Test value 'serac_displacement/l2norms[0]' out of tolerance: baseline value=0.251395613996565, test value=0.251529521894169
ERROR: Test value 'serac_displacement/l2norms[1]' out of tolerance: baseline value=0.498432309185511, test value=0.498413786890275
ERROR: Test value 'serac_displacement/l2norms[2]' out of tolerance: baseline value=0.738551570341719, test value=0.739519844217816
ERROR: Test value 'serac_displacement/l2norms[3]' out of tolerance: baseline value=0.973458524266585, test value=0.97362944877621
ERROR: Test value 'serac_displacement/l2norms[4]' out of tolerance: baseline value=1.19954797985943, test value=1.20124207730023
ERROR: Test value 'serac_displacement/linfnorms[4]' out of tolerance: baseline value=0.628010415343569, test value=0.628049953672262
ERROR: Test value 'serac_displacement/linfnorms[5]' out of tolerance: baseline value=0.741269604240727, test value=0.741356212166571
ERROR: Test value 'serac_displacement/mins[2]' out of tolerance: baseline value=-0.308237598066768, test value=-0.308326825525133
ERROR: Test value 'serac_displacement/mins[3]' out of tolerance: baseline value=-0.369927282496162, test value=-0.370105009461507
ERROR: Test value 'serac_displacement/mins[4]' out of tolerance: baseline value=-0.408031347678698, test value=-0.40826368917436
ERROR: Test value 'serac_displacement/mins[5]' out of tolerance: baseline value=-0.440657227316583, test value=-0.440970696732981
ERROR: Test value 'serac_displacement/maxs[4]' out of tolerance: baseline value=0.590952646439994, test value=0.59099162104609
ERROR: Test value 'serac_displacement/maxs[5]' out of tolerance: baseline value=0.699665018250717, test value=0.699754414583068
Tolerance for serac_acceleration = 1e-05
```

This can be reproduced by checking out the serac branch https://github.com/LLNL/serac/pull/1067

**Comment by btalamini** [comment]
Thanks! This is much easier to see what's going on. There changes are a little larger than the (arbitrary) tolerance, but just barely.

**Review by btalamini — APPROVED** [review]

**Comment by chapman39** [comment]
I updated the baselines of the tests that failed, so that the CI can pass.

**Review by btalamini — COMMENTED** [review]
We have more precise unit tests to check correctness. I think these differences are reasonable for integration tests.

**Comment by btalamini** [comment]
Thanks for digging through this!

File: integration/heat_transfer/static_reaction_exact/static_reaction_exact.ats [hunk 1]
```diff
@@ -5,10 +5,13 @@
 # SPDX-License-Identifier: (BSD-3-Clause)
 ##############################################################################
 
-tolerance_test(name="static_reaction_exact_serial",
-    input_file="{0}/tests/heat_transfer/static_reaction_exact.lua".format(serac_input_files_dir))
+# TODO: Re-enable after 'source' is supported
```
Comment (white238) [inline integration/heat_transfer/static_reaction_exact/static_reaction_exact.ats:8]: Is there an Serac issue for this?

Comment (white238) [reply integration/heat_transfer/static_reaction_exact/static_reaction_exact.ats:8]: im an idiot.. ignore me.

**Review by white238 — APPROVED** [review]

**Pull request merged**